docs/workflows.md

```diff
@@ -57,37 +57,37 @@ Activities are the building blocks of BPMN 2.0, there would be no business proce
 
 #### `End`
 
-In business process modeling, every process has an ending. In BPMN, every path in every process must lead to an end event. Every process starts with a start event, moves through activities and gateways, and then reaches an end. When a business process model reached an end event, it is complete. When a sub process reaches an end event, the process flow returns to the parent process.
+In business process modeling, every process has an ending. In BPMN, every path in every process must lead to an end event. Every process starts with a start event, moves through activities and gateways, and then reaches an end. When a business process model reaches an end event, it is complete. When a subprocess reaches an end event, the process flow returns to the parent process.
 
 ### `Difference`
 
-The main difference between BMPN.io and PlantAnApp Workflows is that in our platform, we create an executable model, not only the visual part (flowcharts). When you create a Workflow, you made a sequence of actions that create behind an entire model, from creating an user and giving him some platform roles to the ability to the ability to create some invoices for a purchase.
+The main difference between BMPN.io and PlantAnApp Workflows is that in our platform, we create an executable model, not only the visual part (flowcharts). When you create a Workflow, you make a sequence of actions that create behind an entire model, from creating a user and giving him some platform roles to the ability to create some invoices for a purchase.
 
-From a technique perspective a Workflow can be considered a reusable method.
+From a technical perspective, a Workflow can be considered a reusable method.
 
 ## `Workflows (Business logic)`
 
 ### `Scope isolation`
 
-Workflows have an isolated scope which means that it will not have access to the context variables from where the workflow was executed and all the values that are required for the workflow to work need to be passed as input parameters.
+A Workflow has an isolated scope which means that it will not have access to the context variables from where the workflow was executed. All the values that are required for the workflow to work need to be passed as input parameters.
```

Comment:
```suggestion
A Workflow has an isolated scope, which means that it will not have access to the context variables from where the workflow was executed. All the values that are required for the workflow to work need to be passed as input parameters.
```